sounddevice.py

```diff
@@ -1942,6 +1971,14 @@ def _hasflag(self, flag):
         """Check a given flag."""
         return bool(self._flags & flag)
 
+    def _updateflag(self, flag, value):
+        """Set the given flag if value is True, otherwise clear the given
+        flag."""
```

Comment:
The "summary" should fit into a single line.

Probably you can shorten it?

What about "Set/reset a given flag."?

This is what `pydocstyle` has to say:

```
sounddevice.py:1974 in private method `_updateflag`:
        D205: 1 blank line required between summary line and description (found 0)
sounddevice.py:1974 in private method `_updateflag`:
        D209: Multi-line docstring closing quotes should be on a separate line
sounddevice.py:1974 in private method `_updateflag`:
        D400: First line should end with a period (not 'n')
```

Reply:
Thanks for this feedback.  I learnt something about docstrings that I didn't know!